whelk-core/src/main/groovy/whelk/component/DependencyCache.groovy

```diff
@@ -49,7 +49,7 @@ class DependencyCache {
             String relation = it[0]
             String iri = it[1]
             dependersCache.invalidate(new Tuple2(iri, relation))
-            dependencyCache.invalidate(new Tuple2(docIri, relation))
+            dependenciesCache.invalidate(new Tuple2(docIri, relation))
```

Comment:
Could we have a key collision here with subject+relation vs. object+relation? (E.g. for `<X> :broader ?o` would we overwrite the cache for `?s :broader <Y>`?)

Reply:
`dependersCache` is `object+relation -> [subject1, ..., subjectN]`
`dependencyCache` is `subject+relation -> [object1, ..., objectN]`

Reply:
When a document is updated only its outgoing links might have changed so we clear out all cached relations where it is the subject.

Reply:
Now I see it! All good then!